app.py

```diff
@@ -134,18 +142,18 @@ async def create_port_webhook():
         "integrationType": "custom",
     }
     response = await send_port_request(
-        "POST",
-        f"webhooks",
-        webhook_data
+        method="POST",
+        endpoint="webhooks",
+        payload=webhook_data
     )
     if isinstance(response, dict):
         status_code = response.get("status_code")
+        error_response = response.get("response")
         if status_code == 442:
             logger.error("Incorrect mapping, kindly fix!")
-            return None
         else:
-            logger.error(f"Error creating Port webhook: {status_code}")
-            return None
+            logger.error(f"Error creating Port webhook: {status_code}, {error_response.text}")
```

Comment:
I see that we're already logging the status code and error message in line 95. So logging this again will create duplication